docs/user/arena-setup.rst

```diff
@@ -2,269 +2,272 @@
-will be warped to the location they joined from, unless the arena has an
-exit warp (optional). Using exit points, it is possible to control
-exactly where players go after a session.
-
-About regions, warps, etc.
---------------------------
-
-Once our arena is built, it's time to set up the MobArena regions,
-warps, and points. Before we do so, let's take a look at what these
-things are and what MobArena uses them for...
-
-**Regions:** An arena needs an *arena region*, and optionally a *lobby
-region*. MobArena uses the arena region to make sure that players aren't
-cheating (by kicking them from the arena if they move outside the
-region), and to make sure that only MobArena's own mobs are spawned
-inside of it. MobArena is extremely xenophobic (afraid of strangers), so
-it tries its best to keep unwanted mobs out of the sessions. Regions are
-set using the Region tools.
-
-**Warps:** When players join a MobArena session, they are teleported to
-the *lobby warp* inside the lobby, where they will *pick their class*
-and ready up using the *ready block* (block of iron). When everyone is
-ready, the players are teleported to the *arena warp* inside of the
-arena. Spectators will be teleported to the *spectator warp* in the
-spectator area, and when players leave an arena, they will either be
-teleported to where they joined from, or to the *exit warp*, if it has
-been set up. Warps are set using the Warps tool.
-
-**Spawnpoints:** The *spawnpoints* of an arena are the locations where
-monsters can spawn from. MobArena will only spawn monsters from
-spawnpoints that have players nearby (in a 15-block radius). Note that
-the number of mobs spawned doesn't depend on how many spawnpoints an
-arena has - the number of mobs is determined by a formula (see
-[[Formulas]]) that only involves the wave number and player count
-(unless you use fixed amounts). Spawnpoints are added using the
-Spawnpoints tool.
-
-**Containers:** The *containers* of an arena are locations of chests,
-dispensers, etc. which contain items that will be renewed when the arena
-ends. Only registered containers will have their contents renewed, so it
-is not enough to simply put a chest in the arena - it also needs to be
-registered using the Chests tool.
+*****************
 
-Setup Mode
+There are four key parts to an arena:
+
+#. Lobby
+#. Arena floor
+#. Spectator area
+#. Exit point
+
+Lobby
+=====
+
+A lobby is where players choose a class before joining an arena. It's also the
+"waiting area" before a new match begins.
+
+Lobbies have two requirements:
+
+- **Class selection signs**: Clicked to select a class
+- **Iron block**: Clicked to mark self as "ready"
+
+Class selection signs must have the name of the class on the first line
+(case-sensitive). The last three lines are not checked and they can have any
+text. You will not receive a confirmation message after making a new sign since
+MobArena does not register the creation of class selection signs.
```

Comment:
Assuming people actually read the docs, I think it is important to retain the note about class sign messages from other plugins. If people place a class sign for, say, "Ranger", and a message pops up saying "class sign registered", it'll be from a different plugin, and chances are the sign will be tampered with by the plugin, resulting in MobArena not actually recognizing it as one if its own.